### PR TITLE
fix: add action buttons to time series column popover 

### LINKED ISSUE #12672: [explore] Time Series Columns Configuration - no save button, (i) wrapped to new line
## Screenshot

Issue 1: No Save Button, however the configuration gets saved is user clicks outside the configuration popup. Is this expected behavior? Given all other settings have save button, it would be good for consistency to have the Save here too.
Screen Shot 2021-01-21 at 5 25 00 PM

Issue 2:
Here the (i) wraps to a new line, consider keeping it inline (make the popup window slightly larger?)
Screen Shot 2021-01-21 at 5 24 29 PM

## Description

Described above alongside screenshot.

## Design input
Confirm if the SAVE button should be on the "Time Series Columns" configuration popup.

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Fixes #12672. 
Improves the layout of the elements.

@rusackas will open another PR to globally fix the tooltip wrap. I increased the spacing to avoid wrapping.

@junlincc @zuzana-vej 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
105433267-27164300-5c0e-11eb-91e5-fb0bd1706089
Screen Shot 2021-05-07 at 10 34 18 AM

### TEST PLAN
1 - Create a Time Series table
2 - Check Times Series Columns control

### ADDITIONAL INFORMATION
- [x] Has associated issue:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/explore/components/controls/BoundsControl.jsx [hunk 1]
```diff
@@ -87,8 +108,8 @@ export default class BoundsControl extends React.Component {
     return (
       <div>
         <ControlHeader {...this.props} />
-        <FormGroup bsSize="small">
-          <Row gutter={16}>
+        <StyledFormGroup bsSize="small">
```
Comment (rusackas) [inline superset-frontend/src/explore/components/controls/BoundsControl.jsx:111]: Hah, you're back to `styled`? :)

File: superset-frontend/src/explore/components/controls/BoundsControl.jsx [hunk 2]
```diff
@@ -45,6 +50,22 @@ export default class BoundsControl extends React.Component {
...
+    if (!_.isEqual(prevProps.value, this.props.value)) {
+      this.update();
+    }
+  }
+
+  update() {
+    this.setState({
+      minMax: [
+        Number.isNaN(this.props.value[0]) ? '' : this.props.value[0],
+        Number.isNaN(this.props.value[1]) ? '' : this.props.value[1],
```
Comment (rusackas) [inline superset-frontend/src/explore/components/controls/BoundsControl.jsx:66]: This behaves a little funny in practice, in that if you type a non-numeric value into one of these fields, it doesn't complain, and the value sticks. However, that non-numeric value IS removed if you type into the _other_ field. 

In other words, you can enter a good value in `min` and then a bad value in `max` - that bad value is only removed if you go back and change `min`

File: superset-frontend/src/explore/components/controls/BoundsControl.jsx (hunk 1 above)
Comment (michael-s-molina) [reply superset-frontend/src/explore/components/controls/BoundsControl.jsx:111]: Yes

File: superset-frontend/src/explore/components/controls/BoundsControl.jsx (hunk 2 above)
Comment (michael-s-molina) [reply superset-frontend/src/explore/components/controls/BoundsControl.jsx:75]: Replaced `Input` by `InputNumber`. Now it only allows numeric values and the user can click on the spinner.

**Comment by junlincc** [comment]
tested locally. LGTM. Thanks again!

**Review by junlin-qa — APPROVED** [review]

**Review by rusackas — APPROVED** [review]
LGTM, thanks!

**Pull request merged**